### PR TITLE
Animal Services, Service Update

### PR DESCRIPTION
**Pull request by martypowell**
- Updated urls for the animal service pages javascript.

### CODE REVIEW

File: js/page-specific/animal-services/adopted-pets.js [hunk 1]
```diff
@@ -1,69 +1,68 @@
...
-  //Utils
-  function getURLParameter(name) {
-    return decodeURI(
-      (RegExp(name + '=' + '(.+?)(&|$)').exec(location.search) || [, null])[1]
-    );
-  }
+	// Utils
+	function getURLParameter(name) {
+		return decodeURI(
+			(RegExp(name + '=' + '(.+?)(&|$)').exec(location.search) || [, null])[1]
```
Comment (mxsnyder) [inline js/page-specific/animal-services/adopted-pets.js]: Might I suggest you use the querystringer utility in /js/utilities?

File: js/page-specific/animal-services/adopted-pets.js [hunk 2]
```diff
@@ -1,69 +1,68 @@
...
+		);
+	}
 
-  function onReady() {
-    var type = getURLParameter('type') === "null" ? "all" : getURLParameter('type');
+	function onReady() {
+		var type = getURLParameter('type') === 'null' ? 'all' : getURLParameter('type');
```
Comment (mxsnyder) [inline js/page-specific/animal-services/adopted-pets.js]: What is this a type of? petType? I suggest a rename to be more specific.

File: js/page-specific/animal-services/adopted-pets.js [hunk 3]
```diff
@@ -1,69 +1,68 @@
...
-      //If there is hash, scroll to that element
-      if (window.location.hash) {
-        document.getElementById(window.location.hash.replace("#", "")).scrollIntoView(true);
-      }
-    });
-  }
+		adoptedPets.Show(type, function () {
+			// If there is hash, scroll to that element
+			if (window.location.hash) {
+				document.getElementById(window.location.hash.replace('#', '')).scrollIntoView(true);
```
Comment (mxsnyder) [inline js/page-specific/animal-services/adopted-pets.js]: Possible exception if `document.getElementById(window.location.hash.replace('#', ''))` is undefined.

File: js/page-specific/animal-services/adopted-pets.js (hunk 1 above)
Comment (martypowell) [reply js/page-specific/animal-services/adopted-pets.js]: I am going to leave this for now, and if we choose to refactor this, I will pull in querystring and make it so there isn't so much repeated code in each of these files.

File: js/page-specific/animal-services/adopted-pets.js (hunk 3 above)
Comment (martypowell) [reply js/page-specific/animal-services/adopted-pets.js]: fixed

**Pull request merged**